src/pudl/extract/ferc1.py

```diff
@@ -213,429 +208,272 @@
 }
 """A mapping of PUDL DB table names to their XBRL and DBF source table names."""
 
-DBF_TYPES = {
-    "C": sa.String,
-    "D": sa.Date,
-    "F": sa.Float,
-    "I": sa.Integer,
-    "L": sa.Boolean,
-    "M": sa.Text,  # 10 digit .DBT block number, stored as a string...
-    "N": sa.Float,
-    "T": sa.DateTime,
-    "0": sa.Integer,  # based on dbf2sqlite mapping
-    "B": "XXX",  # .DBT block number, binary string
-    "@": "XXX",  # Timestamp... Date = Julian Day, Time is in milliseconds?
-    "+": "XXX",  # Autoincrement (e.g. for IDs)
-    "O": "XXX",  # Double, 8 bytes
-    "G": "XXX",  # OLE 10 digit/byte number of a .DBT block, stored as string
-}
-"""dict: A mapping of DBF field types to SQLAlchemy Column types.
 
-This dictionary maps the strings which are used to denote field types in the DBF objects
-to the corresponding generic SQLAlchemy Column types: These definitions come from a
-combination of the dbfread example program dbf2sqlite and this DBF file format
-documentation page: http://www.dbase.com/KnowledgeBase/int/db7_file_fmt.htm
+class FoxProExtractor:
```

Comment:
Seems like `FoxProExtractor` should probably go in the `dbf.py` module?

Reply:
Yes, indeed.